<launch>
  <group ns="droneA">
    <arg name="fcu_url" default="udp://:14551@:14555"/>

    <include file="$(find mavros)/launch/apm.launch">
      <arg name="fcu_url" value="$(arg fcu_url)"/>
    </include>

    <node name="jason_hw_bridge" pkg="jason_ros" type="hw_bridge.py"
      args="-a $(find mas_uav)/config/actions_manifest
      -p $(find mas_uav)/config/perceptions_manifest"
      output="screen"/>

    <node name="python_agent" pkg="mas_uav" type="droneA.py" output="screen"/>

    <node name="jason_comm" pkg="jason_ros_comm" type="comm_udp.py" output="screen">
      <rosparam command="load" file="$(find mas_uav)/params/local_ips.yaml" />
    </node>

  </group>

  <group ns="droneB">
    <arg name="fcu_url" default="udp://:14561@:14565"/>

    <include file="$(find mavros)/launch/apm.launch">
      <arg name="fcu_url" value="$(arg fcu_url)"/>
    </include>

    <node name="jason_hw_bridge" pkg="jason_ros" type="hw_bridge.py"
      args="-a $(find mas_uav)/config/actions_manifest
      -p $(find mas_uav)/config/perceptions_manifest"
      output="screen"/>

    <node name="python_agent" pkg="mas_uav" type="droneB.py" output="screen"/>

    <node name="jason_comm" pkg="jason_ros_comm" type="comm_udp.py" output="screen">
      <rosparam command="load" file="$(find mas_uav)/params/local_ips.yaml" />
    </node>
  </group>
</launch>
